notebooks/minigrid_colab_tutorial.ipynb

```diff
@@ -0,0 +1,558 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e-WlgAw0FDXF"
+      },
+      "source": [
+        "#  Example -  Training DQN agent on Minigrid"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "amrNQCUtqPRw"
+      },
+      "source": [
+        "In RLHive, there are config samples for DQN agents, for different environment including, Atari, Gym, Hanabi, Marlgrid, Minatar, and Minigrid. \n",
```

Comment:
Removing the first comma after "including"

Reply:
It has been removed.